### PR TITLE
Remove useless parentheses

### PR DESCRIPTION
**Pull request by DanielRuf**
### Description (*)

This PR removes a few nore useless parentheses.

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: setup/src/Magento/Setup/Model/Installer.php [hunk 1]
```diff
@@ -925,7 +925,7 @@ private function throwExceptionForNotWritablePaths(array $paths)
      */
     private function handleDBSchemaData($setup, $type, array $request)
     {
-        if (!(($type === 'schema') || ($type === 'data'))) {
+        if (!$type === 'schema') || ($type === 'data')) {
```
Comment (kalpmehta) [inline setup/src/Magento/Setup/Model/Installer.php]: Left parenthesis is missing here

**Review by kalpmehta — CHANGES_REQUESTED** [review]

**Comment by kalpmehta** [comment]
@DanielRuf Please add automated tests

File: setup/src/Magento/Setup/Model/Installer.php (hunk 1 above)
Comment (zhartaunik) [inline setup/src/Magento/Setup/Model/Installer.php]: Operator order is the following: `!$type` converts it to `bool(false)` and further comparison `false` and `'schema'`
This condition `!$type === 'schema'` will always return false

**Comment by DanielRuf** [comment]
> Please add automated tests

For what? I did not add any new logic / code.

File: setup/src/Magento/Setup/Model/Installer.php (hunk 1 above)
Comment (DanielRuf) [reply setup/src/Magento/Setup/Model/Installer.php]: I just forgot to add one `(`, see the other review comment ;-)

I will fix this one.

File: setup/src/Magento/Setup/Model/Installer.php [hunk 2]
```diff
@@ -925,7 +925,7 @@ private function throwExceptionForNotWritablePaths(array $paths)
      */
     private function handleDBSchemaData($setup, $type, array $request)
     {
-        if (!(($type === 'schema') || ($type === 'data'))) {
+        if (!($type === 'schema') || ($type === 'data')) {
```
Comment (kalpmehta) [inline setup/src/Magento/Setup/Model/Installer.php]: @DanielRuf Looks like this changes the existing logic. Can you please review it again? Note the original code is checking IF NOT `(($type === 'schema') || ($type === 'data'))` whereas the new code is checking IF NOT `($type === 'schema')` OR IF TRUE `($type === 'data')`.

**Review by kalpmehta — CHANGES_REQUESTED** [review]

File: setup/src/Magento/Setup/Model/Installer.php (hunk 2 above)
Comment (DanielRuf) [reply setup/src/Magento/Setup/Model/Installer.php]: You are right, I will fix this.

Comment (DanielRuf) [reply setup/src/Magento/Setup/Model/Installer.php]: Should be ok now.

**Review by kalpmehta — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @kalpmehta, thank you for the review.
[ENGCOM-6016](https://jira.corp.magento.com/browse/ENGCOM-6016) has been created to process this Pull Request
:eight_spoked_asterisk: @kalpmehta, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**